### PR TITLE
fix approved

### PR DESCRIPTION
**Pull request by quanglx-noovado**
## Related Tickets:
- [#32127](https://edu-redmine.sun-asterisk.vn/issues/32127)

## WHAT this PR does?
 - Chỉnh sửa phần approved cho admin

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by quanglx-noovado** [comment]
ready

File: resources/views/job_list.blade.php [hunk 1]
```diff
@@ -21,9 +22,9 @@
                                 <div class="tab_grid">
                                     <div class="jobs-item with-thumb">
                                         <div class="thumb">
-                                            <a href="">
-                                                <img src="" class="img-responsive" alt="" />
-                                            </a>
+                                            <a href="{{ route('companies.show', ['company' => $job->company_id]) }}">
+                                                <img src="{{ asset($job->url) }}" class="img-responsive" alt="" />
+                                            </a>
```
Comment (TDM1911) [inline resources/views/job_list.blade.php]:
```suggestion
                                            </a>
```

File: resources/views/job_list.blade.php [hunk 2]
```diff
@@ -4,6 +4,7 @@
 @include('layouts.search')
 <div class="container">
     <div class="single">
+        <div class="col-md-1"></div>
         <div class="col-md-9 single_right">
```
Comment (TDM1911) [inline resources/views/job_list.blade.php]:
```suggestion
        <div class="col-md-9 offset-md-1 single_right">
```

File: app/Http/Controllers/AdminController.php [hunk 3]
```diff
@@ -52,8 +52,16 @@ public function acceptEmployer($id)
...
+        $approveJobs = Job::with(['tags', 'company', 'images'])->where('status', config('job_config.approve'))->get();
+        $unapproveJobs = Job::with(['tags', 'company', 'images'])->where('status', config('job_config.unapprove'))->get();
+
+        foreach ($approveJobs as $job) {
+            $job->url =  $job->images()->where('type', config('user.avatar'))->first()->url;
+        }
+
+        foreach ($unapproveJobs as $job) {
+            $job->url =  $job->images()->where('type', config('user.avatar'))->first()->url;
+        }
```
Comment (huyenttm-0901) [inline app/Http/Controllers/AdminController.php]: nếu không tim thấy frist thì sao em?

**Comment by quanglx-noovado** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
c có câu hỏi

**Comment by quanglx-noovado** [comment]
ready

**Review by congnx-2423 — APPROVED** [review]

**Pull request merged**